view/adminhtml/ui_component/adyen_notificationsoverview.xml

```diff
@@ -0,0 +1,119 @@
+<listing xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
```

Comment:
shall we change the file to adyen_notifications_overview.xml for better readability ?

Reply:
Done